test/es-module/test-esm-type-flag-errors.js

```diff
@@ -26,9 +26,9 @@ expect('--input-type=module', packageTypeModuleMain,
 
 try {
   require('../fixtures/es-modules/package-type-module/index.js');
-  assert.fail('Expected CJS to fail loading from type: module package.');
 } catch (e) {
-  assert(e.toString().match(/Error \[ERR_REQUIRE_ESM\]: Must use import to load ES Module:/));
+  // Verify CommonJS load is attempted but fails on ES module syntax
+  assert(e instanceof SyntaxError);
```

Comment:
This should use [`assert.throws(...)`](https://nodejs.org/dist/latest-v12.x/docs/api/assert.html#assert_assert_throws_fn_error_message), else this will be skipped if it does not throw.

Reply:
Added.